src/config.h

```diff
@@ -33,8 +33,10 @@ struct touchscreen_deadzone {
 };
 
 enum {
-  INPUT_SPECIAL_KEY_MODE = -10,
-  INPUT_SPECIAL_KEY_PAUSE
+  INPUT_SPECIAL_KEY_PAUSE = -0xffff,
```

Comment:
What do you mean? In `special_input` value > 0 is keyboard, < 0 is controller and < 0xf000 is special cases.